x/token/keeper/query_server_test.go

```diff
@@ -0,0 +1,176 @@
+package keeper_test
+
+import (
+	sdk "github.com/line/lbm-sdk/types"
+	"github.com/line/lbm-sdk/types/query"
+	"github.com/line/lbm-sdk/x/token"
+)
+
+func (s *KeeperTestSuite) TestQueryTokenBalance() {
+	testCases := map[string]struct{
+		classId string
+		address sdk.AccAddress
+		valid bool
+		postTest func(res *token.QueryTokenBalanceResponse)
+	}{
+		"valid request": {
```

Comment:
No existence error case.

Reply:
It returns 0 token, instead of emitting an error, following the strategy of lbm-sdk (and cosmos-sdk).